docs/set_up_admin_tails.rst

```diff
@@ -55,25 +55,27 @@ Start by running the following commands to download the git repository.
 Verify the Release Tag
 ~~~~~~~~~~~~~~~~~~~~~~
 
-First, download the *Freedom of the Press Foundation Master Signing Key*
-and verify the fingerprint.
+First, download and verify the *Freedom of the Press Foundation Master
+Signing Key*. When passing the full public key fingerprint to the
+``--recv-key`` command, GPG will implicitly verify that the fingerprint
+of the key received matches the argument passed.
 
 .. code:: sh
 
-    gpg --keyserver pool.sks-keyservers.net --recv-key B89A29DB2128160B8E4B1B4CBADDE0C7FC9F6818
-    gpg --fingerprint B89A29DB2128160B8E4B1B4CBADDE0C7FC9F6818
+    gpg --keyserver hkp://qdigse2yzvuglcix.onion --recv-key B89A29DB2128160B8E4B1B4CBADDE0C7FC9F6818
 
-The Freedom of the Press Foundation Master Signing Key should have a
-fingerprint of ``B89A 29DB 2128 160B 8E4B 1B4C BADD E0C7 FC9F
-6818``.
+It is important you type this out correctly. If you are not copy-pasting
+it, we recommend typing the fingerprint out with spaces like so ``B89A
+29DB 2128 160B 8E4B 1B4C BADD E0C7 FC9F 6818``, double-checking you have
+it correct, and then removing the spaces before pressing "enter."
```

Comment:
I would move the earlier sentence:

> When passing the full public key fingerprint to the ``--recv-key`` command, GPG will implicitly verify that the fingerprint of the key received matches the argument passed.

to be an admonition after this paragraph.